test/test_sax.rb

```diff
@@ -31,6 +31,17 @@ def test_entity_replacement
       assert_equal '--1234--', results[1]
     end
 
+    def test_predefined_entity_replacement
```

Comment:
```suggestion
    def test_characters_predefined_entities
```

And could you rename existing `test_entity_replacement` to `test_characters_custom_entities`?

And could you split character reference case to `test_characters_character_reference`?

Reply:
OK
I see.